dashboard/apps/prototype/management/commands/cache.py

```diff
@@ -25,6 +26,7 @@ def generate(self):
         """
         generate cache
         """
+        cache_tools.METHOD_CACHE_WRITE_ONLY = True
```

Comment:
alternatively, to be defensive, we can use a context manager like this in the command, which will restore the original value when the command finishes.

```
@contextmanager
def preserve_value(value):
    preserved = value
    try:
        yield
    finally:
        value = preserved

with preserve_value(cache_tools.METHOD_CACHE_WRITE_ONLY):
       self.generate()
```